### PR TITLE
[15.0][MIG] account_invoice_date_due: Migration to 15.0

### PR DESCRIPTION
**Pull request by OriolVForgeFlow**
Standard migration
@forgeflow

### CODE REVIEW

File: account_invoice_date_due/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Update Invoice's Due Date",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (JoanMForgeFlow) [inline account_invoice_date_due/__manifest__.py]: Please chage the version to:
"version": "15.0.1.0.0",

**Review by JoanMForgeFlow — CHANGES_REQUESTED** [review]

**Review by JoanMForgeFlow — APPROVED** [review]
Functional review :+1: 
Minor changes

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
Hi @pedrobaeza I think this one is ready for merging too

**Pull request merged**